Editor/UI/Wizard/FirstTime/XRSDKSetupPage.cs

```diff
@@ -15,31 +15,44 @@ internal class XRSDKSetupPage : WizardPage
     {
         private enum XRLoader
         {
-            None,
             OpenVR,
             Oculus,
             WindowsMR,
-            Other
+            Other,
+            None
         }
 
-        private const string OpenVRInfo = "OpenVR XR Plugin will be imported into the project.";
-        private const string OculusInfo = "Oculus XR Plugin will be imported into the project.";
-        private const string WindowsMRInfo = "Windows XR Plugin will be imported into the project.";
+        private readonly List<XRLoader> options = new List<XRLoader>(Enum.GetValues(typeof(XRLoader)).Cast<XRLoader>());
+
+        private readonly List<string> nameplates = new List<string>()
+        {
+            "HTC Vive / Valve Index (OpenVR)",
+            "Oculus Quest/Rift",
+            "Windows MR",
+            "Other",
+            "None"
+        };
+
+        private IDictionary<XRLoader, string> infos = new Dictionary<XRLoader, string>
+        {
+            { XRLoader.OpenVR, "OpenVR XR Plugin will be imported into the project." },
+            { XRLoader.Oculus, "Oculus XR Plugin will be imported into the project." },
+            { XRLoader.WindowsMR, "Windows XR Plugin will be imported into the project." },
+            { XRLoader.Other, "Right now we do not support other than the listed plugins." },
+            { XRLoader.None, "If you dont want to import any XR related plugins, press the skip button." }
+        };
 
         private GUIContent infoContent;
-        private string otherHardwareText;
+
+        [SerializeField]
```

Comment:
Why is this necessary? The instance of the class is never deleted while the wizard is opened.